### PR TITLE
Fix sqlite error

### PR DESCRIPTION
**Pull request by Koochr**
One thing needed to be done is changing node vesrion in .env
Also I'm not sure if I should add changes to defaultConfig to this PR, please check

### CODE REVIEW

**Comment by Koochr** [comment]
Upd: I've removed some useless code, which led to subsquent `findOrCreate` calls, removed some useless `setImmediate`, and I think I've fixed all of arweave and db errors, though sometimes I still see `EPIPE` error

**Review by tankakatan — COMMENTED** [review]
Thanks, all seems good. Please consider the comments on `findById` method family. And also please fix the indentation. It is ruined in some files. Please make sure you're using spaces instead of tabs.

File: db/model.js [hunk 1]
```diff
@@ -49,13 +49,15 @@ class Model extends sequelize_lib.Model {
...
     }
 
-    static async findOrCreate(...args) {
-        if (args.length === 1 && !(typeof args[0] === 'object' && args[0] !== null && !Array.isArray(args[0]))) {
-            const returned = await super.findOrCreate({ where: { id: args[0] }});
-            return returned[0];
-        } else {
-            return await super.findOrCreate(...args);
-        }
+    static async findOrCreate({where, defaults}) {
```
[the remark this answers is not in the document]
Comment (Koochr) [reply db/model.js:52]: Sure, I've seached and replaced all the usages of this function, so it's safe.

File: db/model.js [hunk 2]
```diff
@@ -59,6 +59,10 @@ class Model extends sequelize_lib.Model {
 		    ...defaults
 	    })
     }
+		
+		static async findByIdOrCreate(id, defaults) {
```
Comment (tankakatan) [inline db/model.js:63]: Thanks for adding this, but please fix the indentation.

**Review by tankakatan — APPROVED** [review]

**Review by tankakatan — APPROVED** [review]

**Pull request merged**